src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -27,13 +28,15 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
```

Comment:
With regards to the error message, might want to look into text formatting of the "..." snippet after the tag portion, as the "..." appears after remark as well. 

![image](https://github.com/user-attachments/assets/9b502134-409c-4a52-9c0e-e15502e4dcfa)